hack/opa/policy.rego

```diff
@@ -1,8 +1,24 @@
 package concourse
 
-# replace with 'false' to add rules
-default allow = true
+default decision = {"allowed": true}
```

Comment:
I really like this, I think it's a good starter example for people to work from!